recipes/SimpleElastix/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: example
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kaspermarstal/SimpleElastix.git
+  git_tag: 93b1482100c9fa8fba156b4d8f28f0388debc38e
```

Comment:
As this is equivalent to this tag `v0.9.0-SimpleElastix`, would it be alright if we use that tag as well. Also, this let's you use jinja with the tag too. :wink: